### PR TITLE
Fix broken tests and improve test coverage

### PR DESCRIPTION
**Pull request by debj1t**

### CODE REVIEW

File: Kubernetes/windows/debug/networkmonitor/networkhealth.ps1 [hunk 1]
```diff
@@ -942,7 +942,7 @@ class StaleRemoteEndpoints : DiagnosticTest {
             $stale_remote_endpoint = $true
             if ($endpoint.IsRemoteEndpoint -eq $true) {
                 foreach ($lbPolicy in $lbPolicies) {
-                    if ($endpoint.IsRemoteEndpoint -eq $true -and $lbPolicy.EndpointIpAddresses.Contains($endpoint.IPAddress))
+                if ($lbPolicy.EndpointIpAddresses.Contains($endpoint.IPAddress))
```
Comment (jayanthaMicrosoft) [inline Kubernetes/windows/debug/networkmonitor/networkhealth.ps1:945]: do we need a 4-space indent?

File: Kubernetes/windows/debug/networkmonitor/networkhealth.ps1 [hunk 2]
```diff
@@ -986,8 +986,8 @@ class LoadBalancerStaleRemoteEndpoints : DiagnosticTest {
         }
         $stale_endpoints = [System.Collections.ArrayList] ($lb_endpoints | Select-Object -Unique)
         foreach($endpoint in $endpointsData) {
-            if($stale_endpoints.Contains($endpoint.Identifier)) {
-                $stale_endpoints.remove($endpoint.Identifier)
+            if($stale_endpoints -Contains $endpoint.Identifier) {
+                $stale_endpoints = $stale_endpoints -ne $endpoint.Identifier
```
Comment (jayanthaMicrosoft) [inline Kubernetes/windows/debug/networkmonitor/networkhealth.ps1:990]: is -ne is a comparison operator? Or does it also modify stale_endpoints container?

**Review by jayanthaMicrosoft — COMMENTED** [review]
2 minor comments

File: Kubernetes/windows/debug/networkmonitor/networkhealth.ps1 (hunk 2 above)
Comment (debj1t) [reply Kubernetes/windows/debug/networkmonitor/networkhealth.ps1:990]: yes, and yes.

**Comment by debj1t** [comment]
@daschott This is also ready, waiting for it to be merged.

**Pull request merged**